models/articles.models.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../db/connection');
+
+exports.selectAllArticles = () => {
+  return db.query(
+    `SELECT a.article_id, a.title, a.topic, a.author, a.created_at, a.votes, a.article_img_url, CAST (COUNT (c.comment_id) AS INTEGER) AS comment_count
```

Comment:
Nice work on this query, casting the result as a number is particularly tricky 🙌